book/book.toml

```diff
@@ -0,0 +1,5 @@
+[book]
+authors = ["Polonius authors"]
+multilingual = false
+src = "src"
+title = "Polonius"
```

Comment:
Could the book be build on CI?

Reply:
It could and it should be, but I was hoping to cajole somebody else into doing that :)

Reply:
I'll take care of that